error.go

```diff
@@ -107,6 +107,16 @@ func (nnr NetworkNameError) Error() string {
 // Forbidden denotes the type of this error
 func (nnr NetworkNameError) Forbidden() {}
 
+// PredefinedNetworkExistError is returned when user tries to create predefined network that already exists.
+type PredefinedNetworkExistError string
+
+func (nnr PredefinedNetworkExistError) Error() string {
```

Comment:
convention nit: can the variable `nnr` be named something along the lines of `pnn` to indicate Predefined Network Name? I think `nnr` is used above for NetworkNameError.

Reply:
changed it.